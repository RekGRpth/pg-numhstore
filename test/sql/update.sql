DROP EXTENSION numhstore CASCADE;
CREATE EXTENSION numhstore VERSION '0.1.6';
ALTER EXTENSION numhstore UPDATE TO '0.1.7';