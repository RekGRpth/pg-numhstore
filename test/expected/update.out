DROP EXTENSION numhstore CASCADE;
CREATE EXTENSION numhstore VERSION '0.1.3';
ALTER EXTENSION numhstore UPDATE TO '0.1.4';
